### DISCUSSION TITLE
Integration with Influxdb and Grafana

### DISCUSSION THREAD

**Discussion by stuartmaxwell** [hassio-addons/addon-adguard-home#324] [Ideas]
Hi all, has anyone considered trying to integrate the AdGuard querylog with the InfluxDB addon so that it can be queried by the Grafana addon? I found [some instructions for how to do this with Telegraf](https://github.com/AdguardTeam/AdGuardHome/issues/573#issuecomment-851625990) but this was for a full install of AdGuard Home and I'm not sure how to go about this with the Hassio add-ons and how to share data between them. The first hurdle I've encountered is that I can't even find where the file is stored, I was hoping it would be in `/opt/AdGuard/data/querylog.json` but I can't find it.

Anyone considered this? Thanks.

**Comment by frenck** [comment]
> Hi all, has anyone considered trying to integrate the AdGuard querylog with the InfluxDB addon so that it can be queried by the Grafana addon?

The way that works in the Home Assistant eco system, is adding the using the AdGuard integration in Home Assistant and the InfluxDB integration in Home Assistant. Home Assistant act as an sort of Telegraph in that way.

If you are looking for the way described in your message, the add-on is not a good fit and cannot help you.
